cl/lib/elasticsearch_utils.py

```diff
@@ -68,7 +71,26 @@ def build_fulltext_query(field: str, value: str) -> QueryString | None:
                     f"docketNumber:{docket_number}",
                     value,
                 )
-        return Q("query_string", query=value, fields=[field])
+        if query_type == "query_string":
+            return Q("query_string", query=value, fields=fields)
+
+        elif query_type == "multi_match":
+            q_should = [
+                Q(
+                    "multi_match",
+                    query=value,
+                    fields=fields,
+                    type="cross_fields",
+                    operator="AND",
+                    tie_breaker=0.3,
+                ),
+                Q("query_string", query=value, default_operator="AND"),
+            ]
+            for field in fields:
+                q_should.append(Q("match_phrase", **{field: {"query": value}}))
+
+            return Q("bool", should=q_should)
+
     return None
```

Comment:
Looks like there are two `return` statements here?

Shouldn't this be in an `else` block as the final condition?

And should we return `None` or '[]' ?

Reply:
Yeah, the first return is not needed anymore it was used for Parentheticals search in #2136 but now seems that both can work with the same query, so I removed it.

And the return yes it should be `[]`, changed.

Thank you!